yaml/buildings.yaml

```diff
@@ -18,6 +18,12 @@ globals:
       roof_orientation: {col: "roof:orientation"}
       addr_housenumber: {col: "addr:housenumber"}
       addr_street: {col: "addr:street"}
+  - &building_kind_detail
+      kind_detail:
+        expr: mz_building_kind_detail(building)
+  - &building_part_kind_detail
+      kind_detail:
+        expr: mz_building_kind_detail("building:part")
```

Comment:
Good point, I hadn't considered the different popular values.

Ignoring misspellings, capitalizations, numbers, "y" and "true", here's what's in `building:part` that isn't in `building` in the first 4 pages according to taginfo:

```
apartment
arch
balcony
barracks
base
basement
bell_tower
bridge
bus_station
castle
chimney
church:part
column
corridor
Corridor
default
dome
door
elevator
floor
hall
historic
horizontal
main
museum
offices
parking
part
passageway
pillar
porch
proposed
ramp
room
silos
stairs
stairway
steps
stilobate
technical
tier
vertical
verticalpassage
wall
window
```

Should we just have separate `kind_detail` whitelists for `building` and `building:part`?